packages/selection/src/SelectionContainer.spec.tsx

```diff
@@ -51,8 +51,7 @@ describe('SelectionContainer', () => {
                   item,
                   focusRef: ref,
                   'data-test-id': 'item',
-                  'data-focused': isFocused,
-                  'data-selected': isSelected
```

Comment:
I removed these two attributes in favor of asserting against values from a user's perspective. From a user's perspective, `data-focused` and `data-selected` don't exist.

In replacement of `data-focused`, tests have been updated to assert against `document.activeElement` to ensure the correct item is focused.

In replacement of `data-selected`, tests have been updated to assert against `aria-selected` to ensure the correct item is selected.